github/ci/prow/files/jobs/kubevirt/controller-lifecycle-operator-sdk/controller-lifecycle-operator-sdk-presubmits.yaml

```diff
@@ -0,0 +1,30 @@
+presubmits:
+  kubevirt/controller-lifecycle-operator-sdk:
+  - name: pull-controller-lifecycle-operator-sdk-e2e-k8s-1.18
+    always_run: true
+    optional: false
```

Comment:
Wouldn't it make sense to start out with the lane being optional at first? This is set to always run--so it could potentially block all merges if there's an error we haven't caught yet.

Reply:
Will change that.